src/main/java/org/opentripplanner/middleware/utils/NotificationUtils.java

```diff
@@ -104,24 +136,97 @@ public static VerificationCheck checkSmsVerificationCode(String phoneNumber, Str
         }
     }
 
+    /**
+     * Send notification email to {@link OtpUser}, ensuring the correct from
+     * email address is used (i.e., {@link #FROM_EMAIL}).
+     */
+    public static boolean sendEmail(
+        OtpUser otpUser,
+        String subject,
+        String textTemplate,
+        String htmlTemplate,
+        Object data
+    ) {
+        return sendEmail(FROM_EMAIL, otpUser.email, subject, textTemplate, htmlTemplate, data);
+    }
+
+    /**
+     * Send notification email to {@link AdminUser}, ensuring the correct from
+     * email address is used (i.e., {@link #OTP_ADMIN_DASHBOARD_FROM_EMAIL}).
+     */
+    public static boolean sendEmail(
+        AdminUser adminUser,
+        String subject,
+        String textTemplate,
+        String htmlTemplate,
+        Object data
+    ) {
+        return sendEmail(OTP_ADMIN_DASHBOARD_FROM_EMAIL, adminUser.email, subject, textTemplate, htmlTemplate, data);
+    }
+
+    /**
+     * Send templated email using SparkPost.
+     * @param fromEmail     from email address
+     * @param toEmail       recipient email address
+     * @param subject       email subject liine
+     * @param textTemplate  template to use for email in text format
+     * @param htmlTemplate  template to use for email in HTML format
+     * @param data          template data
+     * @return              whether the email was sent successfully
+     */
+    private static boolean sendEmail(
+        String fromEmail,
+        String toEmail,
+        String subject,
+        String textTemplate,
+        String htmlTemplate,
+        Object data
+    ) {
+        try {
+            String text = TemplateUtils.renderTemplate(textTemplate, data);
+            String html = TemplateUtils.renderTemplate(htmlTemplate, data);
+            return sendEmailViaSparkpost(fromEmail, toEmail, subject, text, html);
+        } catch (TemplateException | IOException e) {
+            // This catch indicates there was an error rendering the template. Note: TemplateUtils#renderTemplate
+            // handles Bugsnag reporting/error logging, so that is not needed here.
```

Comment:
Remove the Bugsnag reference.

Reply:
I don't understand why we would remove the Bugsnag reference? We're using Bugsnag to report the error.

Reply:
This PR is abstracting Bugsnag out, so removing the Bugsnag reference is consistent with that purpose.

Reply:
Bugsnag is still very much a part of the system though. Nothing about Bugsnag reporting has changed and we're still tracking all of the errors in Bugsnag.